fjord/feedback/views.py

```diff
@@ -44,23 +46,39 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    if waffle.flag_is_active(request, 'thankyou'):
-        template = 'feedback/thanks.html'
-        try:
-            opinion_id = request.session.get('opinion_id')
-            user_opinion = Response.objects.get(id=opinion_id)
-        except Response.ObjectDoesNotExist:
-            pass
-        else:
-            word_count = len(user_opinion.description.split())
-            if (user_opinion.locale == 'en-US' and
-                    not user_opinion.happy and
-                    word_count >= 7):
-                template = 'feedback/thanks_sad.html'
-    else:
-        template = 'feedback/thanks.html'
+    suggestions = None
+    response = None
 
-    return render(request, template)
+    if waffle.flag_is_active(request, 'thankyou'):
+        response_id = None
+
+        # If the user is an analyzer/admin, then we let them specify
+        # the response_id via the querystring. This makes debugging
+        # the system easier.
+        if ((request.user.is_authenticated()
+             and request.user.has_perm('analytics.can_view_dashboard'))):
+            response_id = smart_int(request.GET.get('response_id', None))
+
+        # If we don't have a response_id, then pull it from the
+        # session where it was placed if the user had just left
+        # feedback.
+        if response_id is None:
+            response_id = request.session.get('response_id')
+
+        # If we have a response_id, then pull the response and get
+        # suggestions.
+        if response_id is not None:
+            try:
+                response = Response.objects.get(id=response_id)
+            except Response.ObjectDoesNotExist:
+                pass
+            else:
+                suggestions = get_suggestions(response)
+
+    return render(request, 'feedback/thanks.html', {
+        'response': response,
+        'suggestions': suggestions
```

Comment:
There's no suggestion providers, yet.

Basically, I've hit this point where there's a bunch of things I need to do each of which needs thought and they're sort of tangled:

1. build a SUMO Search Suggest API suggest provider
2. create mockups for the new thank you page

So I made these code changes to make it easier to work on those two things in tandem without having different versions of the thanks view.